misk-actions/src/main/kotlin/misk/web/interceptors/LogRequestResponse.kt

```diff
@@ -34,4 +36,6 @@ annotation class LogRequestResponse(
   val errorBodySampling: Double = 0.0,
   /** which deploy environments will not have request/response logging enabled **/
   val excludedEnvironments: Array<String> = [],
+  /** By default log non-error responses **/
+  val enableNonErrorLogging: Boolean = true,
```

Comment:
Consider using an enum instead, i.e. `val loggingLevel: LoggingLevel = LoggingLevel.ALL`, and then have another level like `ERROR_ONLY` which has your equivalent behaviour.

Reply:
Done!